generator.js

```diff
@@ -43,11 +44,21 @@ const processFile = (file, currentPath) => {
 
 const filePath = path.join(__dirname, 'template')
 const projectPath = path.join(process.cwd(), appTitle)
+
 console.log(`Creating ${projectPath} `)
 console.log('Copying over project files')
 processFile('', filePath)
 console.log('Installing NPM Depenedencies')
 execSync('npm install', {cwd: projectPath, stdio: 'inherit'})
+console.log('Initializing a git repository')
+console.log('Making the initial commit !')
```

Comment:
To be consistent with the other messaging just remove the `!` here. `'Making the initial commit'` is fine 👍

Reply:
Cool